Core/src/main/java/com/plotsquared/core/plot/PlotInventory.java

```diff
@@ -103,9 +103,20 @@ public void close() {
         this.open = false;
     }
 
-    public void setItem(int index, PlotItemStack item) {
+    /**
+     * Put an item into this inventory
+     *
+     * @param index the index (= slot) where to place the item
+     * @param item  the item to place
+     * @return {@code true} if the item could be placed, otherwise {@code false}
+     * @see InventoryUtil#setItem(PlotInventory, int, PlotItemStack)
```

Comment:
Missing since tag

Reply:
The method is not new at all, just changed the signature from void to boolean, or should that be commented with the since tag?

Reply:
No, looks like IJ just rendered it weird. Dismiss.